### ISSUE TITLE
LLM policy

### ISSUE THREAD

**Issue by mjskay** [journalovi/journalovi.github.io#22]
Do we need/want a policy on using large language models to help write papers?

**Comment by floe** [comment]
This is actually one thing requested by the library (i.e. our publisher) under their new journal agreement, so we should probably add one soon-ish.

**Comment by chatchavan** [comment]
Can we use the same policy as [ACM's](https://www.acm.org/publications/policies/new-acm-policy-on-authorship)?

> Generative AI tools and technologies, such as ChatGPT, may not be listed as authors of an ACM published Work. The use of generative AI tools and technologies to create content is permitted but must be fully disclosed in the Work. For example, the authors could include the following statement in the Acknowledgements section of the Work: ChatGPT was utilized to generate sections of this Work, including text, tables, graphs, code, data, citations, etc. If you are uncertain ­about the need to disclose the use of a particular tool, err on the side of caution, and include a disclosure in the acknowledgements section of the Work.
>
> Basic word processing systems that recommend and insert replacement text, perform spelling or grammar checks and corrections, or systems that do language translations are to be considered exceptions to this disclosure requirement and are generally permitted and need not be disclosed in the Work. As the line between Generative AI tools and basic word processing systems like MS-Word or Grammarly becomes blurred, this Policy will be updated.

**Comment by floe** [comment]
Yes, good starting point - the CEUR workshop template also has an example for an acknowledgement that I found better than the one in the ACM blurb: "During the preparation of this work, the author(s) used X-GPT-4 and Gramby in order to: Grammar and spelling check. Further, the author(s) used X-AI-IMG for figures 3 and 4 in order to: Generate images. After using these tool(s)/service(s), the author(s) reviewed and edited the content as needed and take(s) full responsibility for the publication’s content."

**Comment by floe** [comment]
OK, I put together a draft in #118, please have a look.
